src/collections/_documentation/cli/releases.md

```diff
@@ -122,7 +122,8 @@ The following options exist to change the behavior of the upload command:
 
 `--strip-prefix` / `--strip-common-prefix`
 
-: When paired with `--rewrite` this will chop-off a prefix from uploaded files. For instance, you can use this to remove a path that is build machine specific. The common prefix version will attempt to automatically guess what the common prefix is and chop that one off automatically.
+: When paired with `--rewrite` this will chop-off a prefix from all sources references inside uploaded source maps. For instance, you can use this to remove a path that is build machine specific. The common prefix version will attempt to automatically guess what the common prefix is and chop that one off automatically.
+This will not modify the uploaded sources paths. To do that, point the upload or upload-sourcemaps command to a more precise directory instead.
```

Comment:
@kamilogorek You will have to put the command names in quotes, as otherwise the unit tests complain about the spelling of "sourcemaps" (which should be "source maps"):

```suggestion
This will not modify the uploaded sources paths. To do that, point the `upload` or `upload-sourcemaps` command to a more precise directory instead.
```